src/CAD/Curve.hh

```diff
@@ -40,7 +40,7 @@ class Curve : public Vec3D, public Shape2D {
  // virtual Vec3D getTangent(double step) = 0; //virtual replaces "abstract" from java
   
   // Overload the << operator
-  friend std::ostream& operator<< (ostream& s, const Curve& c){
+  friend std::ostream& operator<< (std::ostream& s, const Curve& c){
```

Comment:
One of the header files above this (i think `Vec3d.hh`) was pulling in the standard namespace, so the entire project was using it as a result. Removed that and then had to add this in as a fix